assets/js/base/utils/create-notice.ts

```diff
@@ -15,28 +15,19 @@ export const DEFAULT_ERROR_MESSAGE = __(
 	'woo-gutenberg-products-block'
 );
 
-export const hasStoreNoticesContainer = ( container: string ): boolean => {
-	const containers = select( 'wc/store/store-notices' ).getContainers();
-	return containers.includes( container );
+export const getNoticeContexts = () => {
```

Comment:
Pedantic, and I know the name itself is enough of a clue, but does this need a docblock describing it? Skip if not necessary